src/pages/alumni/index.js

```diff
@@ -31,58 +30,40 @@ class Alumni extends Component {
             </h1>
           </div>
         </header>
-        <div className="batch">Batch 2020</div>
+
+        <div className="batch">Batch 2021</div>
         <div className="grid md:grid-cols-2 gap-y-2 sm:grid-cols-1">
-          {this.state.dummy.map((test) => (
-            <Cards
+          {this.state.dummy.map((test, index) => (
+            test.batch == "2017" && <Cards
```

Comment:
No need to hard code the batch year, as it is subject to change. Instead, you can use nested mapping to minimize the code repetition.

See https://github.com/spec-nith/spec-frontend/blob/e19da0b6ed28793e14324d21fb24124170f9d600/src/pages/team/index.js#L33-L46 for more details.